src/components/App.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import Display from './Display';
+import Panel from './Panel';
+import calculate from '../logic/calculate';
+
+export default class App extends React.Component {
```

Comment:
A slicker way to write our classes with the dawn of a neat plugin (`babel-plugin-transform-classes`) and its inclusion to create-react-app:

```javascript
class App extends React.Component {
  state = {
      total: null,
      next: null,
      operation: null,
    };
    
   handleclick = (btn) => this.setState(prevState => calculate(prevState, btn));
}

render(){
 // ditto
 }
```

Reply:
Neat, thanks for letting me know